jwala-integration-test/src/test/java/com/cerner/jwala/ui/selenium/JwalaTestSuite.java

```diff
@@ -36,24 +51,22 @@
         AppDeleteTest.class, JvmOperationsPageDeleteTest.class, WebServerOperationsPageDelete.class, WebServerCreateTest.class,
         WebServerDeleteTest.class, JvmCreateTest.class, JvmDeleteTest.class, GroupDeleteTest.class, BalancerManagerTest.class,
         LogoutTest.class})
-public class JwalaChromeTestSuite extends TestSuite {
+public class JwalaTestSuite extends TestSuite {
 
-    private static final String WEBDRIVER_CHROME_DRIVER = "webdriver.chrome.driver";
     private static final String ELEMENT_SEARCH_RENDER_WAIT_TIME = "element.search.render.wait.time";
+    public static final String WEB_DRIVER = "webDriver";
```

Comment:
Just a very minor thing: should this not be camel case? It looks like it's read as a System Property, but is it ever read from the properties file too?